packages/extension-ui/src/Popup/CreateAccount/CreateAccount.spec.tsx

```diff
@@ -47,20 +47,28 @@ describe('Create Account', () => {
     );
 
   describe('CreateAccount component', () => {
-    const clickNext = () => wrapper.findWhere(
-      (node) => node.type() === 'button' && node.text() === 'Next'
-    ).simulate('click');
+    const clickNext = () =>
+      wrapper.findWhere((node) => node.type() === 'button' && node.text() === 'Next').simulate('click');
 
-    const clickBack = () => wrapper.findWhere(
-      (node) => node.type() === 'button' && !!node.find('.arrowLeft').length
-    ).simulate('click');
+    const clickBack = () =>
+      wrapper.findWhere((node) => node.type() === 'button' && !!node.find('.arrowLeft').length).simulate('click');
 
-    const isPresent = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeTruthy();
-    const isAbsent = <T,>(statelessComponent: FunctionComponent<T>) => expect(wrapper.find(statelessComponent).exists()).toBeFalsy();
+    const isPresent = <T,>(statelessComponent: FunctionComponent<T>) =>
+      expect(wrapper.find(statelessComponent).exists()).toBeTruthy();
+    const isAbsent = <T,>(statelessComponent: FunctionComponent<T>) =>
+      expect(wrapper.find(statelessComponent).exists()).toBeFalsy();
 
-    const checkIsSecretCopiedCheckbox = () => wrapper.find({type: 'checkbox'}).simulate('change', {target: {checked: true}});
+    const checkIsSecretCopiedCheckbox = () =>
+      wrapper.find({ type: 'checkbox' }).simulate('change', { target: { checked: true } });
+
+    // eslint-disable-next-line deprecation/deprecation
+    const isClipboardUntouched = () => expect(document.execCommand).not.toHaveBeenCalled();
```

Comment:
Maybe rather 
```suggestion
    const hasClipboardNotBeenCleared = () => expect(document.execCommand).not.toHaveBeenCalled();
```

as the clipboard _per se_ might have been touched with `navigator.clipboard.writeText`, I presume?